swagger.yml

```diff
@@ -1076,6 +1076,22 @@ paths:
             $ref: '#/definitions/deploy'
         default:
           $ref: '#/responses/error'
+  /deploys/{deploy_id}/ready:
```

Comment:
I'm not sure if it has already been discussed but to me this does not seem very RESTful.

What would you think to use the `update` endpoint `PUT /deploys/{deploy_id}` with a payload of `{state: "READY"}`

we could defer by the payload what to do, and not only seems a bit more RESTful it's a bit more representative of the change we want to do with the resource (Deploy)

I wonder what the rest of the team would think of this as well.

cc. @JGAntunes @kitop @julienp

Reply:
Moving this convo to https://github.com/netlify/bitballoon/pull/16569/files#r1315114299